### ISSUE TITLE
MAST

### ISSUE THREAD

**Issue by sils1297** [coala-analyzer/coala-bears#438]
MAST features some quite cool machine learning tools we could maybe use :)
https://mast-group.github.io/

Migrated from https://github.com/coala-analyzer/coala/issues/2129
area/lintbears

**Comment by Techievena** [comment]
Are we going to implement all the 7 tools in this bear?

**Comment by sils** [comment]
Not necessarily, this needs some thinking first, what makes sense and helps the users most?
